visualization/motion_planning_rviz_plugin/src/motion_planning_frame_planning.cpp

```diff
@@ -55,14 +55,20 @@ void MotionPlanningFrame::planButtonClicked()
 void MotionPlanningFrame::executeButtonClicked()
 {
   ui_->execute_button->setEnabled(false);
-  planning_display_->addBackgroundJob(boost::bind(&MotionPlanningFrame::computeExecuteButtonClicked, this), "execute");
+  boost::thread(boost::bind(&MotionPlanningFrame::computeExecuteButtonClicked, this));
```

Comment:
Why use boost threads when this class already has its own background job framework setup? Seems like a departure of the functional style of the rest of the class.